src/Feature.php

```diff
@@ -62,21 +60,4 @@ protected static function getFacadeAccessor()
     {
         return FeatureManager::class;
     }
-
-    /**
-     * Serialize the given scope for storage.
-     *
-     * @param  mixed  $scope
-     * @return string|null
-     */
-    public static function serializeScope($scope)
-    {
-        return match (true) {
-            $scope === null => '__laravel_null',
-            is_string($scope) => $scope,
-            is_numeric($scope) => (string) $scope,
-            $scope instanceof Model => $scope::class.'|'.$scope->getKey(),
-            default => throw new RuntimeException('Unable to serialize the feature scope to a string. You should implement the FeatureScopeable contract.')
-        };
-    }
```

Comment:
Moved this to the manager class so we don't have to introduce / juggle static state.

I don't see this as a breaking change. `Feature::serializeScope(...)` worked before and continues to work now as it just proxies through to the manager implementation.